services/src/modules/plugins/index.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from 'fs';
```

Comment:
Probably better to just import the promises directly instead of explicitly accessing it later (`import { promises as fs } from 'fs';`), especially since in node 14 it's already a separate import (`import fs from 'fs/promises'`)

Reply:
For some reason `import fs from 'fs/promises'` doesn't work. So I've changed to `import { promises as fs } from 'fs'`

Reply:
Yeah, I tried to change to `lts` but I think that it's still 12.
It works with 14. But I don't want to change because we have 3 more places with the same import.
We'll change it in some upgrade dependencies PR.